Wrapping/Generators/Python/Tests/PythonTemplateTest.py

```diff
@@ -119,6 +119,17 @@
 median_kwarg = itk.MedianImageFilter.New(Input=reader.GetOutput())
 assert itk.class_(median) == itk.class_(median_kwarg)
 
+# filter type determined by the input passed as a primary input name input
+median_primary_kwarg = itk.MedianImageFilter.New(Primary=reader.GetOutput())
```

Comment:
Can you please tell me what type of image is returned by `reader.GetOutput()`? Apparently the reader reads "test.png". Where can I possibly find it?

Reply:
It is defined at the beginning of this file, as UC2:
https://github.com/InsightSoftwareConsortium/ITK/pull/4921/files/c1e867950b3db1df0d7e5ef715f19fe37f3bb83b#diff-a92a445377264af61e33fa7b96ca1f16668040e89ca0a0e35b8b888d6d91b532L23-L40

Reply:
OK, thanks @dzenanz ! And the "test.png" file itself, is it actually being read during the test? Because I cannot find it myself.

Reply:
I have 23 files named `test.png` on my computer, but none is in the right location. Here is the one from `M:\a\ITK-git\Modules\Remote\ParabolicMorphology\test\images\test.png`:
![test](https://github.com/user-attachments/assets/48d15ef4-f7ee-4b1b-98a3-2926cb78f19b)

Reply:
> Can you please tell me what type of image is returned by reader.GetOutput()? Apparently the reader reads "test.png". Where can I possibly find it?

@N-Dekker in Python, the type at runtime can be examined with:

```python
print(type(reader.GetOutput()))
```